src/version.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2021 Emmanuel Engelhart <kelson@kiwix.org>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU  General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301, USA.
+ */
+
+#include <iostream>
+#include <sstream>
+
+#include <version.h>
+#include <zim/zim.h>
+#include <kiwix_config.h>
+#include <unicode/uvernum.h>
```

Comment:
Didn't this header cause problems under Windows?

Reply:
@veloman-yunkan No, the problem with libzim was the libicu should not be included if libzim is not compiled with libxapian (which is the case for Windows CI). That said,  `unicode/uversion.h` seems slightly better `unicode/uvernum.h` but without any impact of any sort (both work fine).